scripts/rel/cut.sh

```diff
@@ -13,14 +13,14 @@ usage() {
     cat <<EOF
 $0 RELEASE_GIT_TAG [option]
 RELEASE_GIT_TAG is a 'v*' or 'e*' tag for example:
-  v5.0.12
-  e5.0.0-beta.6
+  v5.1.1
+  e5.1.0-beta.6
 
 options:
   -h|--help:         Print this usage.
 
   -b|--base:         Specify the current release base branch, can be one of
-                     release-50
+                     release-51
```

Comment:
May be we can just use `release` at this point

Reply:
`release` is not quite search friendly. 
how about `release-candidate`?